### PR TITLE
Update migrate-mongo types for version 8.2

### PR DESCRIPTION
**Pull request by ghost91-**
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/seppevs/migrate-mongo/compare/v8.1.0...v8.2.3
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

File: types/migrate-mongo/index.d.ts [hunk 1]
```diff
@@ -1,8 +1,8 @@
-// Type definitions for migrate-mongo 8.1
+// Type definitions for migrate-mongo 8.2
 // Project: https://github.com/seppevs/migrate-mongo#readme
 // Definitions by: Amit Beckenstein <https://github.com/amitbeck>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// Minimum TypeScript Version: 3.2
+// Minimum TypeScript Version: 4.1
```
Comment (amitbeck) [inline types/migrate-mongo/index.d.ts:5]: What change to the type definitions requires version 4.1 and up?

Comment (ghost91-) [reply types/migrate-mongo/index.d.ts:5]: It’s due to the updated dependency on mongodb. I would need to check the details to be able to tell you exactly, but with typescript < 4.1, there are compile errors.

Comment (amitbeck) [reply types/migrate-mongo/index.d.ts:5]: Thanks for the clarification

**Review by amitbeck — APPROVED** [review]
LGTM

**Comment by amitbeck** [comment]
Ready to merge

**Pull request merged**